tst/EnergyPlus/unit/HVACUnitarySystem.unit.cc

```diff
@@ -723,50 +723,52 @@ TEST_F( ZoneUnitarySystemTest, UnitarySystem_MultispeedPerformance ) {
 	EXPECT_NEAR( UnitarySystem( 1 ).DesignHeatingCapacity, 31732.204, 0.001 );
 	EXPECT_EQ( UnitarySystem( 1 ).DesignHeatingCapacity, VarSpeedCoil( 2 ).RatedCapHeat );
 
-	// Coil:*:DX:VariableSpeed coils DO NOT use multispeed performance object information
-	// when and if this is corrected, these numbers will change based on the inputs for the MS performance object
-	EXPECT_NEAR( UnitarySystem( 1 ).CoolVolumeFlowRate( 1 ), 0.638152, 0.000001 );
+	EXPECT_NEAR( UnitarySystem( 1 ).CoolVolumeFlowRate( 1 ), 0.177264, 0.000001 );
```

Comment:
See how much these changed. Before, the coil object determined what the flow rate would be, Now the MSHP design spec object sets the flow and I set it up as 10% per speed (1.772 * 10% = 0.1772).